### PR TITLE
Create configuration variables which should be used to specify active OAuth providers

### PR DESCRIPTION
**Pull request by ysv**
… OAuth providers

### CODE REVIEW

File: lib/generators/config/templates/application.yml.erb [hunk 1]
```diff
@@ -96,21 +98,16 @@ defaults: &defaults
   HEADER_RUBRIC_BACKGROUND_COLOR: ~
   MAIL_HEADER_BACKGROUND_COLOR:   ~
 
-  # Configuration variables for webhooks.
```
Comment (yivo) [inline lib/generators/config/templates/application.yml.erb]: @yssavchuk Pls rebase.Why have you removed it?

File: lib/generators/config/templates/application.yml.erb [hunk 2]
```diff
@@ -96,21 +98,16 @@ defaults: &defaults
...
-  WEBHOOKS_SECURE_URL_COMPONENT: ~ # Optional. Set value if you want to secure URL to webhooks.
-  # Example:
-  #   When WEBHOOKS_SECURE_URL_COMPONENT is set to "B6p2csgf2h" the URL to webhooks becomes /webhooks/B6p2csgf2h/btc/hook.
-  #   The default URL is /webhooks/btc/hook.
-
 development:
   <<: *defaults
+  # Development oauth2 providers
```
Comment (yivo) [inline lib/generators/config/templates/application.yml.erb]: @yssavchuk Pls ALWAYS put dot at the end of comment and name thing according to their originals – OAuth2. Also lets change google_oauth2 to google. It is OK.

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by mod** [comment]
@yivo is it urgent ?
I would highly favor using a OR
it's auth0 or barong or google
and this code will stay for a while I believe

**Review by yivo — APPROVED** [review]

**Comment by yivo** [comment]
@mod This is what you asked some time ago. It isn't urgent. Do you want to allow selecting only single provider? Or leave as is (separated by comma)?

**Comment by yivo** [comment]
Fixed.

**Pull request merged**